inginious/frontend/templates/course_admin/task_dispensers/section.html

```diff
@@ -2,7 +2,7 @@
 {# more information about the licensing of this file. #}
 
 {% for section in sections %}
-    {% if section.is_empty() %}
+    {% if section.get_tasks() | length ==0  %}
```

Comment:
I fear that doing so has quite the same result as the one deleted for accessibilities : it will recursively navigate and count tasks for each subsection. Maybe `is_empty` remains the best option here.